app/code/Bulbulatory/Recomendations/Setup/InstallData.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Bulbulatory\Recomendations\Setup;
+
+use Magento\Framework\Setup\InstallDataInterface;
+use Magento\Framework\Setup\ModuleContextInterface;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+
+class InstallData implements InstallDataInterface
+{
+	protected $date;
+
+	public function __construct(\Magento\Framework\Stdlib\DateTime\DateTime $date)
+	{
+		$this->date = $date;
+	}
+
+	public function install(ModuleDataSetupInterface $setup, ModuleContextInterface $context)
+	{
+		$dataNewsRows = [
```

Comment:
Tutaj, jako że jest to fikcyjny projekt, dodanie wierszy do bazy nie jest problemem, ale w przypadku prawdziwych projektów nie powinniśmy nic dodawać do tabel w bazie, powinny one być uzupełniane poprzez akcje wykonywane przez użytkowników. Tutaj do testów jak najbardziej może zostać, ale taka uwaga na przyszłość.